include/glow/Runtime/HostManager/HostManager.h

```diff
@@ -43,6 +43,20 @@ class Provisioner;
 /// handles DeviceManager initialization, houses the Executor, and calls into
 /// the Partitioner and Provisioner for network initialization.
 class HostManager final {
+  /// NetworkData contains data about each network in HostManager that is needed
+  /// by the runtime.
+  struct NetworkData {
+    DAG dag;
+    // list of placeholders owned by the NetworkData.
+    PlaceholderList placeholders;
+
```

Comment:
So why constants won't be needed in this struct?

Reply:
Collect constants will copy them
https://github.com/pytorch/glow/blob/master/include/glow/Backends/BackendUtils.h#L70-L77